web/src/pages/team/tmoorma.md

```diff
@@ -44,7 +44,10 @@ Hardware
 - EE Shop: At the beginning of the week, I visited the EE shop on behalf of the team to look for hall sensors that the team could use for feasibility testing. I was able to get two Hall effect sensors which were the exact model the team had been looking at previously. I also looked for any other components that may be useful for prototyping, but did not find anything else.
 - Design Discussion: During the mandatory lab on 8/30/2023, Alan, Ben, and I discussed the design of the daughter PCBs. Our discussion primarily focused on how the Hall effect sensors and LEDs should be laid out and how data will flow between the microcontrollers. My main contribution to the discussion was suggesting one level of the microcontrollers should convert the hall effect sensor readings to a digital signal based on a threshold. The location of the magnet could then be determined by averaging the position of all Hall effect sensors that have been "triggered", rather than more complex computations based on the Hall effect sensor's magnitude readings. This would decrease the amount of data that would need to be transferred between the microcontrollers. Alan and Ben are both interested in exploring this method further, with Ben proposing that we include multiple thresholds to increase the resolution while still compressing the data. Feasibility testing with the Hall effect sensors will give us a better idea on whether this method of data compression will work.
 
-![W2m pic](/477grp5/team/trevor/Trevor_Week1_Figure2c.PNG)
+<img src="/477grp5/team/trevor/Trevor_Week1_Figure2a.PNG" width="40%">
+<img src="/477grp5/team/trevor/Trevor_Week1_Figure2b.PNG" width="40%">
+<img src="/477grp5/team/trevor/Trevor_Week1_Figure2c.PNG" width="40%">
+<img src="/477grp5/team/trevor/Trevor_Week1_Figure2d.PNG" width="40%">
```

Comment:
https://gist.github.com/trusktr/93175b620d47827ffdedbf52433e3b37

This doesn't make a grid

Reply:
Fixed